draft-ietf-quic-recovery.md

```diff
@@ -226,24 +226,25 @@ QUIC supports many ACK ranges, opposed to TCP's 3 SACK ranges.  In high loss
 environments, this speeds recovery, reduces spurious retransmits, and ensures
 forward progress without relying on timeouts.
 
-### Explicit Correction For Delayed ACKs
+### Explicit Correction For Delayed ACKs {#ack-delay}
```

Comment:
Use words.  ACKs are delayed in the network, because they are frames.  Acknowledgments are the things that you put in the frame (and the things where we need to compensate for delays).

```suggestion
### Explicit Correction For Delayed Acknowledgements {#ack-delay}
```